src/registry/addressNL/edit.stories.ts

```diff
@@ -35,10 +35,14 @@ export const PostcodeValidationTabWithoutConfiguration: Story = {
     await step('Navigate to validation tab and open Postcode validation', async () => {
       await userEvent.click(canvas.getByRole('link', {name: 'Validation'}));
       await userEvent.click(canvas.getAllByText('Postcode')[0]);
-      expect(await canvas.findByLabelText('Regular expression for postcode')).toBeVisible();
-      expect(await canvas.findByText('NL')).toBeVisible();
-      expect(await canvas.findByText('EN')).toBeVisible();
-      expect(await canvas.findByText('Error code')).toBeVisible();
+
+      await waitFor(async () => {
```

Comment:
it's a bit strange that here the `waitFor` is needed, since the `await canvas.findByLabelText(...)` is already async and uses `waitFor` under the hood :thinking: 

I'd like to better understand if/what is going wrong in these interaction tests - often an actual root cause can be found.

Reply:
Without the `waitFor` they keep failing locally..

Maybe this is again one of those situations that only fail locally, and that the CI will work fine..